devolving_music/lib/song_scores.py

```diff
@@ -29,18 +31,35 @@ def get_scores(self) -> "dict[int, ScoreSuite]":
 
         return self.song_score_dict
 
+    def get_scores_list(self) -> "list[ScoreSuite]":
+        # calculates scores for all submissions using current comparisons
+        # returns list of all score suite objects
+        return list(self.get_scores().values())
+
     def get_compare_submission_random(self, submission_id) -> SongSubmission:
         key_list = list(self.song_score_dict.keys())
         key_list.remove(submission_id)
-        # once you have a critical number of comparisons then pull from quality
-        # list
-        return self.song_score_dict.get(random.choice(key_list)).song_submission
+        return self.song_score_dict.get(
+            random.choice(key_list)).song_submission
+
+    def get_compare_submission_closest(self, score_suite_obj: "ScoreSuite", information_threshold : int, update_scores = False):
```

Comment:
```suggestion
    def get_compare_submission_closest(self, score_suite: "ScoreSuite", information_threshold : int, update_scores = False):
```

There's no reason to suffix a variable name with `_obj`. Everything is an object in Python.

Reply:
I know I like this because I won't mistakenly confuse score suite for the class score_suite. It makes it very clear it is the object of the class and not the class.